### DISCUSSION TITLE
PLC not work :(

### DISCUSSION THREAD

**Discussion by domgggg** [MikaylaFischler/cc-mek-scada#436] [Support] [answered]
The plc seems to not connect to the network. I updated the installer and the plc to the latest version but nothing on the monitors or plc is showing readings to the reactor or reactor plc being online
(also idk how to put images on these messages so once you respond ill reply with the screenshots I took)

**Comment by MikaylaFischler** [comment]
Yeah I'll need more info than that, such as screenshots you mentioned when you get the chance and/or the state of the PLC's NETWORK and STATUS lights.

I added this page recently, so I'll link it in case this is your problem: https://github.com/MikaylaFischler/cc-mek-scada/wiki/FAQ-and-Common-Problems#my-computers-wont-connect-to-each-other

**Reply by domgggg** [reply]
![Screenshot 2024-02-28 173838](https://github.com/MikaylaFischler/cc-mek-scada/assets/160686612/3945ec3d-c194-4929-9262-2a51db79c3a7)
![Screenshot 2024-02-28 173738](https://github.com/MikaylaFischler/cc-mek-scada/assets/160686612/b90b0a2e-c7f9-47df-98be-f60f569c5d09)
![Screenshot 2024-02-28 173711](https://github.com/MikaylaFischler/cc-mek-scada/assets/160686612/45c8ec4c-6739-4284-8736-310f21c64463)

It just seems like the reactor wont connect to the reactor plc even though its connected to the network.
Also let me know if their is any other problems you see on the monitors because I'm pretty sure I don't have everything hooked up right, I'm still trying to figure a lot of things out.

Thank you

**Reply by MikaylaFischler** [reply]
The red STATUS light indicates the PLC didn't fully start up and is waiting on peripherals, which in this case is elaborated on by the red REACTOR light indicating that there isn't a reactor connected. Is the PLC touching a Fission Reactor Logic Adapter or connected via a wired modem to one?

**Reply by domgggg** [reply]
Oh I connected using a wired modem to a reactor port.

**Reply by domgggg** [reply]
also theirs more problems still.
 
Every time I try to start the reactor from the monitor it says something about the units and it wont let me start it. Im just confused about the units.

**Reply by MikaylaFischler** [reply, marked as answer]
You can't run auto control mode until you assign a reactor unit to one of the non-manual assignments (primary, secondary, etc). You may find this page helpful: https://github.com/MikaylaFischler/cc-mek-scada/wiki/Operating-Procedure

**Reply by domgggg** [reply]
okay thank you al problems are fixed.
